README.md

```diff
@@ -34,8 +34,9 @@ By default, the text `Jenkins` is displayed. You can choose any text you like or
 Logo and logo text form a hyperlink that gets you back to the Jenkins root page (similar to the original header).
 
 ## The title
-Optionally an additional title can be shown after a separator. This can contain html (sanitized with owasp), e.g. to 
-include a link to some other site. Css already takes care to make the link properly styled.
+Optionally an additional title can be shown after a separator. The title is formatted using the global markup
+formatter (see <code>Manage Jenkins > Security > Markup Formatter</code>) and may contain html, e.g. to include a link
```

Comment:
That's a bit misleading when you write `and may contain html`. If the plain text formatter is active then you can't include html.